### ISSUE TITLE
Forms/Fields in a template? 

### ISSUE THREAD

**Issue by TMS-Namespace** [zadam/trilium#2544]
Hi, 
I wonder if it is possible to create template that will contain forms/fields that can be re-used.

An example for this usage case,  is a contact list,  so for example some fields will contain only name, other,  address,  etc… 

The purpose of this is not searching,  as in usual databases,  but to keep info structured (contact list is just a simple example,  not really my usage case). 

If that is possible,  can you please advise the api that can be used (I didn't  manage to find trilium api reference) ? And how to specify that a field is a plain text box,  while the other is a rich one,  that will display the default trilium formatting toolbar (if this possible of course).

**Comment by zadam** [comment]
Hi, did you check [promoted atttributes](https://github.com/zadam/trilium/wiki/Promoted-attributes). It seems to fit some of the requirements you're looking for.

**Comment by TMS-Namespace** [comment]
> Hi, did you check [promoted atttributes](https://github.com/zadam/trilium/wiki/Promoted-attributes)? It seems to fit some of the requirements you're looking for.

Yes,  great,  thank you,  this indeed partially covers what I asked about. 

So,  as I understand,  there will be no possibility to add something like 'rich text" attribute?  Since they are prompted and crowded in the top of the document,  so multi-line rich textbox will take lots of space? 

Or we can customize how those fields are arranged via css maybe? 

Then should I  make a feature request for a multi-line rich text box? 

Thank you again.

**Comment by zadam** [comment]
Yes, multiline rich text promoted attribute is not supported (every text note has ofc. the "main" rich text content).

Creating a more targeted feature request for this functionality would make sense, yes. Although I'm not sure if it fits into the current concept.

**Comment by TMS-Namespace** [comment]
OK, thx again,

I actually found that using sub notes, with a viewType attribute as a list for the parent note, dose what I mainly wanted to do, with "rich text" attributes.

However, I faced two issues now, for the first one I opened a bug report, since I believe it is conceptually wrong, since export/print ignores all prompted attributes.

Second one, that exporting/printing, again, will ignore the sub notes. Is it possible somehow to export/print a note, such that the result visually, will be very similar to what we see inside the application itself? I mean with all sub notes included as a list?

**Comment by zadam** [comment]
> Is it possible somehow to export/print a note, such that the result visually, will be very similar to what we see inside the application itself? I mean with all sub notes included as a list? ("this note and all of its descendants" exports children as a separate files).

No, not really. Printing is quite specific in that many elements are displayed differently or not at all, so there needs to be specific support for that ... which there isn't.
